### PR TITLE
chore: Daily Habit Copy Updates

### PR DESCRIPTION
**Pull request by calebpanza**
## ✏️ Description
We need to be able to support custom typography for certain aspects of Daily Habits. This PR introduces a couple of copy changes to the default copy and also enables customization for some areas.

1. Changes the default copy of the IDLE Hero Card title
2. Changes the default copy IN PROGRESS Hero Card title
3. Changes the default copy of the Today Gratitude title
4. Changes the default copy of the Today Prayer Card title
5. Enables prop overrides on the Today Gratitude Card
6. Enables prop overrides on the Today Prayer Card
7. Enables prop overrides on the Today Reading Card

## 🔬 To Test
1. Open up the `mobile/theme.json`
2. Update the file with the JSON provided below
3. Run the app and navigate to the Daily tab
4. Adjust different copy for the `daily-habits.TodayGratitudeCard`, `daily-habits.TodayPrayerCard`, and `daily-habits.TodayReadingCard` to see the changes _(note: you might need to rebuild the app)_

```json
{
  "colors": {},
  "typography": {},
  "overrides": {
    "daily-habits.TodayGratitudeCard": {
      "title": "Did this work?",
      "summary": "YES IT DID BABAAYYY!!!!"
    }
  }
}

```

## 📸 Screenshots
Screenshot_20221216-140533

## ⚠️ To-do Before Merge
@conrad-vanl needs to approve of this before it can get merged

## 📝 Checklist

- [x] I have performed a self-review of my code
- [x] I have tested my code works in the project (nothing breaks)
- [x] I have commented my code in hard-to-understand areas
- [x] New and existing tests and linting pass locally with my changes
- [x] I have made corresponding changes to the documentation (new envs?)
- [x] For PR's with design updates/changes, I have consulted with the product designer and made all necessary adjustments in Figma

### CODE REVIEW

File: templates/mobile/src/components/DailyHabitsTab/HeroCard/IdleCard.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ IdleCard.propTypes = {
   subtitle: PropTypes.string,
 };
 IdleCard.defaultProps = {
-  title: 'Boost your daily habits.',
+  title: 'Practive your daily habits now',
```
Comment (dsilva609) [inline templates/mobile/src/components/DailyHabitsTab/HeroCard/IdleCard.js:54]: Practice?

Comment (calebpanza) [reply templates/mobile/src/components/DailyHabitsTab/HeroCard/IdleCard.js]: Thank you!

**Comment by conrad-vanl** [comment]
Can we change the "See what others are saying"? Even if it was as simple as "Others are _____" that might help

**Comment by calebpanza** [comment]
I'll give you one better! Defaults are updated _and_ can be adjusted per church with overrides

**Review by conrad-vanl — APPROVED** [review]

**Pull request merged**